src/pyupgrade/plugins/unnecessary_lru_cache_params.rs

```diff
@@ -1,14 +1,21 @@
 use rustpython_parser::ast::Expr;
 
 use crate::check_ast::Checker;
-use crate::pyupgrade::checks;
+use crate::pyupgrade::{checks, fixes};
 
 pub fn unnecessary_lru_cache_params(checker: &mut Checker, decorator_list: &[Expr]) {
-    if let Some(check) = checks::unnecessary_lru_cache_params(
+    if let Some(mut check) = checks::unnecessary_lru_cache_params(
         decorator_list,
         checker.settings.target_version,
         checker.from_imports.get("functools"),
     ) {
+        if checker.patch() {
+            if let Some(fix) =
+                fixes::remove_unnecessary_lur_cache_params(checker.locator, check.location)
```

Comment:
Can you add `UnnecessaryLRUCacheParams` to the `fixable` list in `src/checks.rs`?

Reply:
It's already in the `fixable` list right ?

Reply:
Oh right! It was my mistake to mark it as `fixable` in the previous PR then.